include/jwt-cpp/jwt.h

```diff
@@ -543,28 +543,21 @@ namespace jwt {
 		}
 
 		/**
-		 * \brief Convert the certificate provided as base64 DER to PEM.
-		 *
-		 * This is useful when using with JWKs as x5c claim is encoded as base64 DER. More info
-		 * (here)[https://tools.ietf.org/html/rfc7517#section-4.7]
+		 * \brief Convert the certificate provided as DER to PEM.
 		 *
-		 * \tparam Decode is callabled, taking a string_type and returns a string_type.
-		 * It should ensure the padding of the input and then base64 decode and return
-		 * the results.
+		 * \tparam Any is used to stop linker errors instead of using inline.
```

Comment:
Whats the reasoning behind using a dummy template arg instead of inline directly ?
Given the comment I suspect you had a reasoning for the decision, I am curious what that was.

Reply:
The two reasons which are mostly preference. Firstly, imo its more likely to be interpreted incorrectly by a reader. Secondly, an unnecessary `inline` can maybe screw with the compiler so Instead of taking the risk I use a substitute.

However, I think consistency should be prioritized.